region/crateria/central/Parlor and Alcatraz.json

```diff
@@ -391,7 +391,7 @@
       "name": "G-Mode Setup - Speed Through the Blocks then Get Hit By Geemer",
```

Comment:
Seems weird to have these strat components combined, to me.
strat 1 - comeInGettingBlueSpeed to reach the hub.
strat 2- G-Mode setup using speed from the hub.